publishable/database/migrations/2017_11_26_014010_drop_permission_groups_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DropPermissionGroupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('permissions', function (Blueprint $table) {
+            $table->dropColumn('permission_group_id');
```

Comment:
Were we using this field/table at all? I don't remember it getting implemented.  If not, is this change related to the primary purpose of this PR, or is it an unrelated change? If the latter, it should be moved to it's own PR

Reply:
I think Permissions group was added for handling User Multi Roles, but was never implemented.
With this PR, I don't see the use of Group permissions, so I remove it.

Reply:
Fair enough.  Works for me.

Reply:
I could see another use case for `permission_group_id`. If permissions were grouped they could be mass assigned or removed from a role, while also being able to add and remove individual permissions. AWS works similarly, where you can mass assign permissions to a user, or select from groups of permissions. That was you can give a user full access to let's say S3, without having to pick each individual S3 permission.

Reply:
@whyayala, that's what roles are. Additional grouping for the sole purpose of assignment is just added complexity

Reply:
@fletch3555 I understand that, but if you have a lot of users, roles, and permissions it could become encumbering for an admin to manage permission to role assignments on a granular level. It's a minor convenience and I agree that it adds more complexity, but I felt obligated to mention it.

Reply:
I completely understand and appreciate that obligation, but for now at least, I don't see the benefit outweighing the cost of complexity, so we're going to hold off on that.  If the argument gets made later, we'll consider adding it back as part of that implementation.  For now, it does nothing.

Thank you.